packages/connect-node-test/src/node-readme.spec.ts

```diff
@@ -60,34 +61,53 @@ describe("node readme", function () {
     },
   } as const;
 
+  const optionsHttp2 = {
+    baseUrl: "https://demo.connect.build",
+    httpVersion: "2" as const,
+  };
+  const optionsHttp1 = {
+    baseUrl: "https://demo.connect.build",
+    httpVersion: "1.1" as const,
+  };
+
   it("createConnectTransport()", async function () {
     // A transport for clients using the gRPC protocol with Node.js `http` module
-    const transport = createConnectTransport({
-      baseUrl: "https://demo.connect.build",
-      httpVersion: "1.1",
-    });
+    const transport = createConnectTransport(optionsHttp1);
```

Comment:
This spec file tests that the examples we give in https://www.npmjs.com/package/@bufbuild/connect-node work.

Can you leave the transport options verbatim?

Reply:
sure :) 0f0adcd7e3e78c254af3b531bf0b361b7ce2d9ba